### PR TITLE
fixed typos and rephrased for clarity

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
## Explanation

This PR includes changes to correct typos and reword certain sentences for better understanding. The changes aim to improve the clarity and readability of the affected sections

## What type of PR is this
 
documentation

### CODE REVIEW

File: docs/develop/build-and-run/cli.md [hunk 1]
```diff
@@ -30,18 +30,18 @@ If users install the WasmEdge from the install script with the option `-e tf,ima
   * The `wasmedge` tool with TensorFlow-Lite, and `wasmedge-image` extensions.
   * Only on `x86_64` and `aarch64` Linux platforms, Android, and `x86_64` MacOS.
 
-The `wasmedge` CLI tool will execute the WebAssembly in ahead-of-time(AOT) mode and interpreter mode. It depends on if you use `wasmedgec` to AoT compile the wasm file.
+The `wasmedge` CLI tool will execute the WebAssembly in ahead-of-time(AOT) mode and interpreter mode if you use `wasmedgec` to AoT compile the wasm file.
```
Comment (alabulei1) [inline docs/develop/build-and-run/cli.md:33]: if we u use `wasmedgec` to AoT compile the wasm file, then WasmEdge is in the AoT mode.

Comment (Mackenzie-OO7) [reply docs/develop/build-and-run/cli.md:33]: Okay, thanks for pointing that out!

**Review by Swatishree-Mahapatra — APPROVED** [review]
Looks good to me!

**Review by Bhardwaj-Himanshu — APPROVED** [review]

File: docs/develop/build-and-run/cli.md [hunk 2]
```diff
@@ -30,18 +30,18 @@ If users install the WasmEdge from the install script with the option `-e tf,ima
   * The `wasmedge` tool with TensorFlow-Lite, and `wasmedge-image` extensions.
   * Only on `x86_64` and `aarch64` Linux platforms, Android, and `x86_64` MacOS.
 
-The `wasmedge` CLI tool will execute the WebAssembly in ahead-of-time(AOT) mode and interpreter mode. It depends on if you use `wasmedgec` to AoT compile the wasm file.
+The `wasmedge` CLI tool will execute the wasm file in ahead-of-time(AOT) mode and interpreter mode. If the file has been compiled with `wasmedgec`, then WasmEdge will execute it in AOT mode, otherwise WasmEdge will execute it in interpreter mode.
```
[the remark this answers is not in the document]
Comment (Bhardwaj-Himanshu) [reply docs/develop/build-and-run/cli.md]: I hope you have run the `git pull upstream` to sort out some conflicts.

[the remark this answers is not in the document]
Comment (Mackenzie-OO7) [reply docs/develop/build-and-run/cli.md]: Thank You! I have done that now.

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
LGTM. Thanks, and sorry for my late reply. @Mackenzie-OO7, @Bhardwaj-Himanshu and @Swatishree-Mahapatra

**Pull request merged**